tasks/ubuntu_repositories.yml

```diff
@@ -1,6 +1,18 @@
 ---
 - name: Add Ubuntu repositories
+  become: true
+  # Use the builtin Ubuntu capability for adding universe, within which wireguard lives
+  command: "add-apt-repository universe"
+  register: adduniverserepo
```

Comment:
Where is this used?

Reply:
On the following line:
`  changed_when: "'already enabled' not in adduniverserepo.stdout"`

Reply:
Sometimes I wonder if I'm blind :)